oauthdoctor/diag/config.go

```diff
@@ -536,15 +538,23 @@ func (c *ConfigFile) Validate() (bool, error) {
 
 // MinGoVersion test for the minimum version of Go required.
```

Comment:
Add  test saying that we require a minimum of 1.11 because we are using modules and include a link to the Wiki.

Reply:
Do you mean unit tests? It's already added.

Reply:
Fat fingers. That should be text.